modules/pel/peltool/peltool.py

```diff
@@ -179,6 +179,28 @@ def buildOutput(sections: list, out: OrderedDict):
             counts[name][1] = modifier + 1
 
 
+def prettyPrint(Mdata: str, desiredSpace: int = 34) -> str:
+    # After index of these 2 characters ":  need to add desired space.
+    CHARACTER_SPACE = 2
+    lines = Mdata.split("\n")
+    for i in range(len(lines)):
+        line = lines[i]
+        if( "\":" in line and "{" not in line ):
```

Comment:
nit:  python doesn't require ( ) on if statements
also, looks like less than 4 space indenting? (though hard to tell because of these comments.)

Reply:
Will try avoiding usage of Parenthesis in If conditions.
There were 4 space indentation in the code.

Reply:
please fix.

also, i don't think the indentation is right: https://github.com/harsh-agarwal1/openpower-pel-parsers/blob/align_json/modules/pel/peltool/peltool.py#L188

Reply:
Addressed the issue in the latest commit.